plugins/riemann-controller/riemann_controller/config.py

```diff
@@ -27,11 +27,13 @@ def create(ctx, policy_types, policy_triggers, groups, config_template):
                 'group': group_name,
                 'policy': policy_name,
                 'policy_type': policy['type'],
+                'policy_properties': policy['properties'],
```

Comment:
Policy properties are already injected as the top level `template_vars`, there is no need to add them again into the metadata